docs/porting/libraries.md

```diff
@@ -1,5 +1,311 @@
+
+### Sandboxing
+**Why was it discontinued?** Sandboxing, i.e. relying on the runtime or the framework to constrain which resources a managed application can access, is considered a non-goal for .NET Core. Sandboxing applications and components is also really hard to get right, which is we recommend you not to rely on it. It also makes the implementation more complicated and often negatively affects performance of applications that don’t use sandboxing. Hence, we do not offer sandboxing features in .NET Core.
+
+**What should you use instead?** Use operating system provided security boundaries, such as user accounts for running processes with the least set of privileges.
+
+To learn about all of the discontinued tech, read [Unsupported Technologies](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/porting.md#unsupported-technologies),
+
+## Targeting the .NET Platform Standard
+
+The first thing to understand about porting a library to .NET Core is that you must target a version of the [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md).  In fact, it's accurate to say that targeting .NET Core means targeting a version of the .NET Platform Standard.
+
+Long story short, this means that you'll have to make a tradeoff between APIs you can use and platforms you can support, and pick the version of the .NET Platform standard that best suits the tradeoff you wish to make.
+
+As of right now, there are 6 different versions to consider: .NET Platform Standard 1.0 through 1.5.  If you pick a higher version, you get access to more APIs at the cost of running on less targets.  If you pick a lower version, your code can run on more targets but at the cost of less APIs available to you.
+
+A key thing to understand is that **a project targeting a lower version cannot reference a project targeting a higher version**.  For example, a project targeting the .NET Platform Standard version 1.2 cannot reference projects that target .NET Platform Standard version 1.3 or higher.  Projects **can** reference lower versions, though, so a project targeting .NET Platform Standard 1.3 can reference a project targeting .NET Platform Standard 1.2 or lower.
+
+It's recommended that you pick a single .NET Platform Standard version and use that throughout your library.
+
+Read more in [The .NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md).
+
+## .NET Core project model overview
+
+.NET Core introduces a new project model that you may wish to spend some time understanding before you begin porting your code.  Rather than provide a comparison with the .NET Framework project model, this section covers the essential pieces.
+
+### The project file: `project.json`
```

Comment:
I chose not to mention it here because `project.json` isn't tied to either of the Visual Studio project systems.  Not sure I like point to it at this point, because that introduces nuances in Visual Studio that don't have anything to do with learning how things work in .NET Core.